diff -urN libkgapi-2.2.0/libkgapi2/CMakeLists.txt libkgapi-2.2.0-patched/libkgapi2/CMakeLists.txt
--- libkgapi-2.2.0/libkgapi2/CMakeLists.txt	2014-07-22 00:47:21.000000000 +1100
+++ libkgapi-2.2.0-patched/libkgapi2/CMakeLists.txt	2014-09-30 23:55:49.414072567 +1100
@@ -351,6 +351,11 @@
 install(FILES ${LibKGAPI_tasks_HDRS} DESTINATION ${INCLUDE_INSTALL_DIR}/libkgapi2/tasks)
 install(FILES ${LibKGAPI_ui_HDRS} DESTINATION ${INCLUDE_INSTALL_DIR}/libkgapi2/ui)
 
+if(NOT WIN32)
+    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/libkgapi.pc.cmake ${CMAKE_CURRENT_BINARY_DIR}/libkgapi.pc)
+    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libkgapi.pc DESTINATION ${LIB_INSTALL_DIR}/pkgconfig)
+endif(NOT WIN32)
+
 # === CMake stuff
 
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/LibKGAPI2Config.cmake.in
diff -urN libkgapi-2.2.0/libkgapi2/libkgapi.pc.cmake libkgapi-2.2.0-patched/libkgapi2/libkgapi.pc.cmake
--- libkgapi-2.2.0/libkgapi2/libkgapi.pc.cmake	1970-01-01 10:00:00.000000000 +1000
+++ libkgapi-2.2.0-patched/libkgapi2/libkgapi.pc.cmake	2014-09-30 23:55:49.414072567 +1100
@@ -0,0 +1,12 @@
+prefix=${CMAKE_INSTALL_PREFIX}
+exec_prefix=${BIN_INSTALL_DIR}
+libdir=${LIB_INSTALL_DIR}
+includedir=${INCLUDE_INSTALL_DIR}
+
+Name: libkgapi
+Description: A C++ library for accessing Google services.
+URL: http://projects.kde.org/libkgapi
+Requires:
+Version: ${KGAPI_VERSION}
+Libs: -L${LIB_INSTALL_DIR} -lkgapi2
+Cflags: -I${INCLUDE_INSTALL_DIR}
